packages/manager/apps/dedicated/client/app/billing/payment/method/add/routing.js

```diff
@@ -105,14 +105,21 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
         $transition$,
         $translate,
         goPaymentList,
-      ) => () =>
-        goPaymentList(
+      ) => () => {
+        const { callbackUrl } = $transition$.params();
+        if (callbackUrl) {
+          window.location.href = callbackUrl;
```

Comment:
not sure if it's a security issue but i'm wondering if we should validate that the host of callbackUrl belongs to OVHcloud